build-tools/dl_parent_devfiles.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Path of stacks directory in the registry
+STACKS_DIR=/registry/stacks
```

Comment:
Need this to override `STACKS_DIR`:
```suggestion
STACKS_DIR=${STACKS_DIR:-/registry/stacks}
```